SpatialGDK/Documentation/features.md

```diff
@@ -50,9 +50,7 @@ Actor property handover is a new feature we’re introducing with the GDK. It re
 
 ## Cross-server RPCs
 
-`Cross-server RPC`s are a new type of RPC that functions between two servers. These enable a server without authority over an Actor to communicate with the server which has authority and is able to manipulate the Actor. For more information, see documentation.
-
-[//]: # (TODO: Add link to the Cross-server RPC doc)
+`Cross-server RPC`s are a new type of RPC that functions between two servers. These enable a server without authority over an Actor to communicate with the server which has authority and is able to manipulate the Actor. For more information see the documentation on [Cross-server RPCs]({{urlRoot}}/content/cross-server-rpcs).
```

Comment:
glossary link to authority. (Put this in backticks so you can see the actual link)
`[authority]({{urlRoot}}/content/glossary#authority)`

Reply:
"These enable a server without [authority]({{urlRoot}}/content/glossary#authority) over an Actor to communicate with the server which has authority and is able to manipulate the Actor."

-> The sentence is hard to follow if you don't already know what this means. It assumes that you know you need worker authority to work in a multiserver environment and one of the problems of worker authority is cross-server communication. They may know this but we can probably explain it more outside in.
I think the key is; any Actor can affect a change over any other Actors, regardless of which server has authority over them.

Suggest...

"As SpatialOS has multiple servers, only one server has the [authority]({{urlRoot}}/content/glossary#authority) to change an Actor at any one time. However, as actions by Actors under the authority of different servers need to affect change over each other, cross-server RPC allows this, regardless of what server has authority over an Actor."

Reply:
?? Not sure if it's clearer. :-)

Reply:
As SpatialOS has multiple servers, only one server has the [authority]({{urlRoot}}/content/glossary#authority) to manipulate an Actor at any one time. Cross-server RPCs enable a server without authority over an Actor to tell the server which has authority over that Actor to manipulate it. For more information see the documentation on [Cross-server RPCs]({{urlRoot}}/content/cross-server-rpcs).